vite.config.ts

```diff
@@ -6,6 +6,9 @@ import vue from "@vitejs/plugin-vue";
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [vue()],
+  server: {
+    https: true,
```

Comment:
is auto redirect naa https eenvoudig te regelen> anders svp in de readme opnemen.
svp ook even aangeven dat je de openldap optie in dex moet gebruiken en dat je incognito niet kan gebruiken (of misschien wel met deze instructies https://stackoverflow.com/a/63587751)

Reply:
Dat lijkt me niet eenvoudig. De server draait maar op 1 poort, je kan denk ik niet redirecten naar dezelfde poort met een ander protocol. Als je de dev server runt, komt er in de terminal een link naar de dev server. die link klopt, want is https. Is dat niet voldoende?

Tweede punt neem ik op in de readme.